Demos/DemoLightbox/Podfile

```diff
@@ -3,6 +3,9 @@ platform :ios, '8.0'
 use_frameworks!
 inhibit_all_warnings!
 
-pod 'Lightbox', path: '../../'
-pod 'Sugar'
-pod 'Hue', git: 'https://github.com/hyperoslo/Hue'
+target 'DemoLightbox' do 
+	pod 'Lightbox', path: '../../'
+	pod 'Sugar'
+	pod 'Hue', git: 'https://github.com/hyperoslo/Hue'
```

Comment:
My point was, it would be better if Lightbox has minimum dependencies. For example `LightboxController` needs `Sugar` and `Hue` and thats what i think we should avoid.